packages/vitest/src/runtime/runners/benchmark.ts

```diff
@@ -135,16 +137,6 @@ async function runBenchmarkSuite(suite: Suite, runner: NodeBenchmarkRunner) {
     suite.result!.duration = performance.now() - start
     suite.result!.state = 'pass'
 
-    tasks
```

Comment:
Did you move the sorting? Why is it removed from here?

Reply:
It's moved to main thread's side, so that we don't need to report tasks twice as `'pass'`.

- Created failing test case first: https://github.com/vitest-dev/vitest/pull/7019/commits/d4223fd55f46c9ef34d0e82326da8e523a110d3b
- Then moved sorting to the other side of RPC: https://github.com/vitest-dev/vitest/pull/7019/commits/9ed93b56752033409c4146c5e1a69c0d94281830

There's `test/benchmark/test/reporter.test.ts` that fails if this sorting and `result.benchmark.rank` setting is wrong.